src/main/java/com/example/bank/auditing/AuditorEntity.java

```diff
@@ -32,7 +32,6 @@ public abstract class AuditorEntity {
     @LastModifiedBy
     private Integer updatedBy;
     @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = localDateTimePattern)
-    /** Jakim cudem będziemy mieć dostęp do tych pól po usunięciu obiektu z bazy?? */
     private LocalDateTime deletedOn;
```

Comment:
W jaki sposób będziemy mieć dostęp do tych pól po usunięciu obiektu z bazy??

Reply:
Klasa (encja) Account rozszerza AuditorEntity. O ile po utworzeniu/edycji konta mamy dostęp do dodanych pól o czasie utworzenia i edycji, to jak po usunięciu konta z bazy danych będziemy mieć dostęp do usuniętego konta, aby sprawdzić pola deletedOn i deletedBy? (Dopuszczam, że baza danych usuwa rekord lub oznacza go jako usunięty /niewidoczny/.) 
Tzn. usuwamy konto z bazy danych. Teoretycznie aktualizują się pola deletedOn i deltedBy, jak sprawdzić ich stan?

Reply:
Gdzie klasa Account rozszerza ta klase o ktorej piszesz? 
<img width="491" alt="image" src="https://github.com/rychcicki/bank-app/assets/112213953/8c3671a9-781d-435b-8259-78fcdb268891">

Reply:
Słusznie. Powinno być klasa Client rozszerza AuditorEntity...

Reply:
To juz predzej. 
Jezeli w auditor entity sa pola jak deletedOn i deletedBy to oczywistym jest ze nie mozesz tego rekordu usunac z bazy danych bo wtedy logicznym jest ze nie bedziesz mial dostepu do tych wartosci. 
Majac pola tego typu w auditorEntity musisz wtedy robic softDelete danego rekordu.
Czyli oznaczac w encji jako usuniety ale nie usuwac go z bazy.